week6/assignment2_3/main.cpp

```diff
@@ -0,0 +1,12 @@
+#include "includes/creature.hpp"
+
+int main() {
+    Creature human('E');
+    Creature robo(102);
+    Creature alien(588.123f);
+    Creature creature;  // Default constructor makes human type
+    std::string check_type{creature.check_creature_type()};
```

Comment:
This is a bit redundant, as you could call the function dirctly in the std::cout as well. 🙂